scripts/prepare_sector_network.py

```diff
@@ -1521,6 +1521,22 @@ def add_heat(n, costs):
                 capital_cost=costs.at[key, 'efficiency'] * costs.at[key, 'fixed'],
                 lifetime=costs.at[key, 'lifetime']
             )
+            
+            if options["biomass_boiler"] and name not in ["urban central"]:
+                #TODO: Add surcharge for pellets
```

Comment:
is this TODO still open?

Reply:
Yes it is, but not so urgent in my mind. A surcharge improves cost accuracy but this tech plays a very minor role in my runs so far anyway. I'll add the pelletisation cost to technology data and refer to it here when I get there.